### ISSUE TITLE
Change Mobile Color Scheme

### ISSUE THREAD

**Issue by atakanyasar** [bounswe/bounswe2024group1#484]
# 📌 Implement a Feature
## 🏡 Component 
- Mobile

## 📚 Description
Change Mobile color schema with the same as current web schema for accessibility.

## 📋 Acceptance Criteria
- [ ] Mobile schema should be the same as web.

## 📃 Related requirements
[Link to section](https://github.com/bounswe/bounswe2024group1/wiki/Requirements#34-accessibility)
> 3.4. Accessibility
> 3.4.1. The web system shall implement the WCAG 2.2 standard to ensure accessibility.
> 3.4.2. This includes but is not limited to:
> 3.4.2.1. The System shall provide aria labels for all icon buttons.
> 3.4.2.2. The System shall provide tabbable inputs and forms.
> 3.4.2.3. The System shall adopt a reasonable hierarchy for screen reader navigation.

### 📅 Deadline
- **Estimated hours of work**: 4 hours
- **Implementation Deadline:** November 12
- **Review Deadline:** November 12  

## 👁‍🗨 Reviewer
@mmtftr

**Comment by mmtftr** [comment]
Closed by #497 Lab 6 -- adjusted all primary colors to use our primary
